tools/RAiDER/cli/raider.py

```diff
@@ -300,10 +300,14 @@ def main(iargs=None):
         # Write the delays to file
         # Different options depending on the inputs
 
-        if not los.ray_trace() and not los.is_Zenith():
+        if los.is_Projected():
             out_filename = w.replace("_ztd", "_std")
-        else:
+            f = f.replace("_ztd", "_std")
+        elif los.ray_trace():
             out_filename = w.replace("_ztd", "_ray")
+            f = f.replace("_ztd", "_ray")
```

Comment:
Ray tracing is still a std. the method should likely be tracked differently than this extension of the filename.

Reply:
This is only for avoiding deleting files when running both rather than for tracking.

Reply:
I would just want to avoid introducing new terms. the commonly accepted terms are zenith and slant. Be it that we have two flavors for slant. How do we currently write out the files?
e.g. MODEL_YYYYMMDDTXXXX_ztd.extension?

Reply:
That's how we write them out yes, `_ztd` / `_std`/ `_ray`